app/views/posts/main_page/_latest_news.html.slim

```diff
@@ -0,0 +1,9 @@
+h3 Latest
+table
+  thead
+    = render 'posts/main_page/table_head', post: @post
+  tbody
+    - @posts.each do |post|
+      - if post.published_date.to_date == DateTime.now.to_date
```

Comment:
This could be moved to helper method.

Reply:
Moreover, you filter the collection by `if` inside a loop, you can use scopes. 
With a help of scope, defined for Post model, you can filter out records with today's date